src/main/java/frc/robot/RobotContainer.java

```diff
@@ -210,90 +210,100 @@ public void scheduleAutonomous() {
 
         // this.driveSubsystem.setThresholds(translationalThreshold, rotationalThreshold);
 
-        // Get and display the selected autonomous mode.
-        AutonomousRoutine autonomousRoutine = autonomousSelector.getSelected();
-        Alliance alliance = allianceSelector.getSelected();
-
-        // Update the robot`s subsystems to be configured for the selected autonomous routine.
-        driveSubsystem.setConfigs();
-        upperAssembly.setRobotInScoringPositionSupplier(driveSubsystem::getInScorePose);
-        upperAssembly.setRobotInIntakingPositionSupplier(driveSubsystem::getInIntakePose);
-        if (upperAssembly instanceof NarwhalUpperAssembly) {
-            ((NarwhalUpperAssembly)upperAssembly).setCanRaiseLiftSupplier(driveSubsystem::getNarwhalCanRaiseLift);
-        }
-
-        Pose2d startingPose;
-
-        // Determine the starting position for the specified autonomous routine and alliance.
-        switch (autonomousRoutine) {
-            case LEFT:
-                startingPose = StartingPosition.LEFT.getPose(alliance);
-                break;
-            case CENTER:
-                startingPose = StartingPosition.CENTER.getPose(alliance);
-                break;
-            case RIGHT:
-                startingPose = StartingPosition.RIGHT.getPose(alliance);
-                break;
-            default:
-                System.err.println("[System]: No alliance starting position selected!");
-                startingPose = null;
-                break;
-        }
+        long startTime = System.currentTimeMillis();
```

Comment:
Not sure if GitHub is just rendering it wrong, but can you fix all the indentation here? It's making the whole file looked modified?